server/app/controllers/checks_controller.py

```diff
@@ -9,12 +13,64 @@
 from app.utils.scrape import scrape_MOSS_report
 
 from datetime import datetime
+from collections import OrderedDict
 
 
 class ChecksController:
+
+    @staticmethod
```

Comment:
This class has all static methods.  That may be indicative of bad design, not using object orientation.  We should discuss this.